### DISCUSSION TITLE
problem with the previous, next button to be responsive

### DISCUSSION THREAD

**Discussion by dlaflamme1** [liedekef/events-made-easy#209] [General]
I have problems to customize the button to be fully responsive for all device 
i did a video to help you understand the problem: https://www.loom.com/share/d8c67854f6a84d96b909d9779d2e9f7e
when on desktop its ok the button are at the good place and in 414 px too but when people with smaller device go to the website the buttons are not centered and not at the good place what is the best way to put this button fully responsive? 
https://palmes2023.davidlaflamme.ca/evenements/
thanks in advance

**Comment by liedekef** [comment]
If using fixed pixels margins and offset, it will never be ok on all devices. I would consider working with grids to be able to place things correctly in that case but I'm not the best CSS guy in the world. EME provides you with the possibility to style everything to your liking, but the styling itself is beyond the scope of EME itself, but a CSS course :-)
